community/kernel/src/main/java/org/neo4j/kernel/impl/store/DynamicArrayStore.java

```diff
@@ -106,7 +113,7 @@ public DynamicArrayStore(
     {
         super( fileName, configuration, idType, idGeneratorFactory, pageCache,
                 logProvider, TYPE_DESCRIPTOR, dataSizeFromConfiguration, recordFormats.dynamic(), recordFormats.storeVersion(), openOptions );
-        allowStorePoints = recordFormats.hasCapability( Capability.POINT_PROPERTIES );
+        allowStorePointsAndTemporal = recordFormats.hasCapability( Capability.POINT_PROPERTIES );
```

Comment:
did you forget to also add `recordFormats.hasCapability( Capability.TEMPORAL_PROPERTIES )`?

Reply:
Indeed